query-planner-js/src/config.ts

```diff
@@ -118,6 +118,11 @@ export type QueryPlannerConfig = {
    * If you aren't aware of this flag, you probably don't need it.
    */
     typeConditionedFetching?: boolean,
+    
+    /**
+     * The maximum time to wait for query planning to complete before giving up and throwing an exception. Defaults to 2 minutes
```

Comment:
You should mention in the docstring here that this configuration option is in milliseconds.